drivers/builtin/include/mbedtls/platform.h

```diff
@@ -446,6 +446,32 @@ mbedtls_platform_context;
 #include "platform_alt.h"
 #endif /* !MBEDTLS_PLATFORM_SETUP_TEARDOWN_ALT */
 
+#if defined(MBEDTLS_PLATFORM_GET_ENTROPY_ALT)
+/**
+ * \brief       User defined callback function that is used from the entropy
+ *              module to gather entropy data from some hardware device.
+ *              The function's prototype is only available when
+ *              `MBEDTLS_PLATFORM_GET_ENTROPY_ALT` is defined.
+ *
+ * \param[out] output           Output buffer where the entropy data will be
+ *                              stored.
+ * \param[in] output_size       Size of the \p output buffer in bytes.
+ * \param[out] output_len       Amount of bytes effectively written in the
+ *                              \p output buffer.
+ * \param[out] entropy_content  Measure of the entropy content (in bits) of the
+ *                              data written in the \p output buffer.
+ *
```

Comment:
Please document the possible return values: 0 for success, `MBEDTLS_ERR_ENTROPY_SOURCE_FAILED` or some other negative value on error.